static/js/release_binaries.js

```diff
@@ -0,0 +1,52 @@
+
+["os", "arch", "version"].map(function(val){
+    eventListener(val);
+});
+
+function eventListener(kind) {
+    let buttonGroupQuery = '#' + kind + ' > .button';
+    let buttonGroup = document.querySelectorAll(buttonGroupQuery);
+
+    let rowsQuery = 'tbody tr';
```

Comment:
Is this narrow enough?

Reply:
> Is this narrow enough?

Narrow in terms of either width or filter criteria? Could you please provide more details?

Reply:
Is this _specific_ enough as a filter / selector?